ansible/roles/chaos/templates/burnio.j2

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Script for BurnIO Chaos
+#
+# based on https://github.com/Netflix/SimianArmy/blob/master/src/main/resources/scripts/burnio
```

Comment:
This link needs an .sh at the end. Also would be nice to have a commit hash instead of master since master changes where it points to